tests/state_test.go

```diff
@@ -75,7 +77,7 @@ func TestState(t *testing.T) {
 	if *testEVM != "" || *testEWASM != "" {
 		// Berlin tests are not expected to pass for external EVMs, yet.
 		//
-		st.skipFork("^Berlin$") // ETH
+		st.skipFork("Berlin")   // ETH
```

Comment:
These `skips` need removed; support provided by EVMCv10.

Reply:
Does it cover both hera and evmone?

Reply:
Yes. EVMOne is EVM, Hera is EWASM.